src/vite.rs

```diff
@@ -229,8 +231,9 @@ impl std::error::Error for ViteError {
     }
 }
 
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Deserialize, Clone)]
 struct ManifestEntry {
     file: String,
+    integrity: Option<String>,
```

Comment:
Should we parse this value from the manifest at `file`?

Reply:
Also, wonder if this should be read at compile time?

Reply:
Hi @mjhoy,

> Should we parse this value from the manifest at `file`?

Sorry, I don't quite understand this part. If I'm not mistaken, `file` should be just a string... I'm not sure how to add `integrity` there, unless you meant to calculate the integrity hash on load/compile time?

---
  
> Also, wonder if this should be read at compile time?

Hmmm, my only concern will be for projects that uses CI/CD, since this behavior will enforce them to build the Vite project first before building the Rust project... where currently you can build either first and it will be all-ok.

Though, I imagine embedding the manifest file during build will improve the startup time, especially when the manifest file is quite big.

What do you think? I'm not sure how to embed/parse file on build but I can try to Google it first.

Thanks!

Reply:
Ah whoops, I was reading this wrong, apologies. Actually, no need to parse at compile, because we're already parsing the manifest at runtime anyway. This change looks good 👍 Can we add a test that exercises it? Also, would you like to add an entry to the `CHANGELOG` in the unreleased section?

Reply:
actually @chez14 I just realized I don't have a test for manifest parsing at all, so no need to add that if you don't want to since it'll be a little bit of setup (i'll try to add it eventually).

Reply:
Ok, I'll use this opportunity to learn Rust better too, hopefully the test is not too messy :sweat_smile:.